tests/ui/rust-2024/safe-outside-extern-with-feature.stderr

```diff
@@ -0,0 +1,14 @@
+error: items cannot be declared with `safe` safety qualifier
```

Comment:
```suggestion
error: items outside of `extern unsafe { }` cannot be declared with `safe` safety qualifier
```

Reply:
I'm not sure how to do { } inside fluent, tried some ways to escape but couldn't really find out.

Reply:
Interesting {"{ }"}.